monai/metrics/occlusion_sensitivity.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import Sequence
+from typing import Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+try:
+    from tqdm import trange
+except (ImportError, AttributeError):
+    trange = range
+
+
+def compute_occlusion_sensitivity(
+    model: nn.Module,
+    image: torch.Tensor,
+    label: Union[int, torch.Tensor],
+    pad_val: float = 0.0,
+    margin: Union[int, Sequence] = 2,
+    n_batch: int = 128,
+) -> np.ndarray:
+    """
+    This function computes the occlusion sensitivity for a model's prediction
+    of a given image. The result is given as ``baseline`` (the probability of
+    a certain output) minus the probability of the output with the occluded
+    area. Therefore, in the output image, higher values mean there was a
+    greater the drop in certainty, indicating the occluded region was more
+    important in the decision process.
+
+    By occlusion sensitivity, we mean how the probability of a given
+    prediction changes as the occluded section of an image changes. This can
+    be useful to understand why a network is making certain decisions.
+
+    Args:
+        model: model to use for inference
```

Comment:
could you specify the expected shape for the model input and output and the labels? if it supports 3D, would be great to have a test case

Reply:
or maybe it's GitHub's issue https://www.githubstatus.com/incidents/k46g5dn2c4lt ...